modules/tinymce/src/models/dom/main/ts/table/api/TableResizeHandler.ts

```diff
@@ -121,10 +121,6 @@ export const TableResizeHandler = (editor: Editor): TableResizeHandler => {
     tableResize.on((sz) => {
       sz.destroy();
     });
-
-    resizeWire.on((w) => {
-      TableWire.remove(editor, w);
-    });
```

Comment:
It's perhaps a bit hard to follow in a PR thread, but why isn't this needed anymore?